briskedu/brisk-learning
# Don't expand docs unlikely to be owned by teacher

If a doc's sole owners are the viewer or a group of teachers as defined in Google classroom, we shouldn't auto-expand to show the number of revisions since it's unlikely it's relevant in those cases.

Hopefully we can find more proxies for student work through the API (at least in the Google classroom case)

Test Plan:
1. Confirm student work expands
2. Confirm teacher docs don't
